common/locales/en/quests.json

```diff
@@ -10,7 +10,13 @@
     "questDetails": "Quest Details",
     "invitations": "Invitations",
     "completed": "Completed!",
+    "rewardsAllParticipants": "Rewards for all Quest Participants",
+    "rewardsQuestOwner": "Additional Rewards for Quest Owner",
+    "questOwnerReceived": "The Quest Owner has also Received",
```

Comment:
Originally they were all capitalized as I based them on the image in #2715. I can update this to be lowercase though